insights/tests/client/connection/test_test_connection.py

```diff
@@ -0,0 +1,175 @@
+from logging import ERROR
```

Comment:
Instead of importing in this object, could you please use `logging.ERROR` in the tests? There are several similarly-sounding variables and it's not clear from the context that this one is a logger level (not everyone is familiar with the `caplog` API).

Reply:
Sure thing! Probably importing just `pyrest` for `mark.parametrize` and similar would make sense too.